docs/develop/scripting-fundamentals.md

```diff
@@ -218,6 +218,120 @@ function main(workbook: ExcelScript.Workbook) {
 }
 ```
 
+### Definining types 
```

Comment:
This whole section doesn't feel like it belongs in "Script Fundamentals". It's far more technical the rest of the article, which centers around creating tables and such. Perhaps is should be its own article? Or maybe there's a larger article like "TypeScript Limitations in Office Scripts" that would include this information.